### ISSUE TITLE
dnspython version 2.0.0 breaks script

### ISSUE THREAD

**Issue by anders-larsson** [Trellmor/bind-adblock#26]
Hello,

It appears changes are needed to support dnspython 2.0.0. It's probably related to `Rdata is now immutable. Use dns.rdata.Rdata.replace() to make a new Rdata based on an existing one.`[1]

Error:
```Traceback (most recent call last):
  File "/opt/bin/update-zonefile.py", line 263, in <module>
    update_serial(zone)
  File "/opt/bin/update-zonefile.py", line 224, in update_serial
    soa.serial += 1
  File "/usr/lib/python3.7/site-packages/dns/rdata.py", line 129, in __setattr__
    raise TypeError("object doesn't support attribute assignment")
TypeError: object doesn't support attribute assignment
```
[1]: https://dnspython.readthedocs.io/en/stable/whatsnew.html

**Comment by Trellmor** [comment]
This has been fixed in a1ad502. You are probably using an old version, try updating.

**Comment by anders-larsson** [comment]
Oh thanks!
